rtsp.go

```diff
@@ -27,18 +31,17 @@ import (
 	"go.viam.com/rdk/resource"
 )
 
+/*
+#include <libavcodec/avcodec.h>
+*/
+import "C"
```

Comment:
Could I recommend keeping this file pure go & having all CGO-specific code contained in the same place (like decoder.go or maybe a new file call codec.go)?